src/e2e_test/vizier/planner/BUILD.bazel

```diff
@@ -24,8 +24,6 @@ pl_go_test(
     ],
     tags = [
         "no_asan",
-        "no_gcc",
-        "no_libcpp",
```

Comment:
Did you check if it works with libcpp?

Reply:
`bazel build` (without specifying a `--config`) couldn't find this test target until `the no_libcpp` was removed. Therefore I thought using libcpp was the default, however, I've tested that `--config clang` results in a successful build as well.